extensions/README.md

```diff
@@ -67,32 +67,35 @@ All glTF object properties (see [glTFProperty.schema.json](../specification/2.0/
 Extensions can't remove existing glTF properties or redefine existing glTF properties to mean something else.
 
 Examples include:
-* **New properties**: `KHR_binary_glTF` introduces a `bufferView` property for shaders, e.g.,
+* **New properties**: `KHR_texture_transform` introduces a set of texture transformation properties, e.g.,
 ```json
 {
-  "shaders": {
-    "a_shader": {
+  "materials": [{
+    "emissiveTexture": {
+      "index": 0,
       "extensions": {
-        "KHR_binary_glTF": {
-          "bufferView": "a_shader_bufferView"
+        "KHR_texture_transform": {
+          "offset": [0, 1],
+          "rotation": 1.57079632679,
+          "scale": [0.5, 0.5]
         }
       }
     }
-  }
+  }]
 }
 ```
 * **New parameter semantics**: `CESIUM_RTC` introduces the `CESIUM_RTC_MODELVIEW` semantic.
-* **Reserved IDs**: `KHR_binary_glTF` introduces an explicitly named `buffer` called `binary_glTF`.
+* **Reserved IDs (glTF 1.0 only)**: `KHR_binary_glTF` introduces an explicitly named `buffer` called `binary_glTF`.
 * **New container formats**: `KHR_binary_glTF` introduces a binary format that contains the glTF JSON and geometry, textures, etc.
```

Comment:
The 2nd and 3rd bullets both appear to be 1.0-only. And is `CESIUM_RTC` a current extension? All of this seems outdated. We could remove these three bullets and replace them with more general wording, like:

> Extensions may add new properties and values, such as attribute semantics or texture mime types. In all Khronos (KHR) extensions, and as best practice for vendor extensions, these feature additions are designed to allow safe fallback consumption in tools that do not recognize an extension in the `extensionsUsed` array.